CHANGES.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
-- [IMPROVED] Add HTTP status code to `Response` objects.
+- [IMPROVED] Added parameter pagination option for views. See `ViewRequest.getResponse(String)`.
+- [IMPROVED] Added HTTP status code to `Response` objects.
```

Comment:
I've made the `CHANGES.md` correction in master and reverted it here in 3f529fb. Everything should be fine after rebase/merge.